### DISCUSSION TITLE
Viewing Photospheres (360 images) with Google Cardboard?

### DISCUSSION THREAD

**Discussion by Aaron-E-Walsh** [mozilla/hubs#2884] [General Questions & Discussion] [answered]
Hi all. 

We have classrooms using Google Cardboard and just started to use Hubs -- love it! But with Cardboard we're having some problems "entering" photospheres (360 images) so that our students can view them properly. Is there a way to step into the middle of a Hubs photospheres when using Google Cardboard?

Aaron

**Comment by cvanmeurs** [comment, marked as answer]
You could probably place a waypoint in the exact center of your photosphere.  But how to access it from a cardboard might be tricky.  I've never tried Hubs on a Cardboard.  How do you move around at all?

**Reply by Aaron-E-Walsh** [reply]
Thanks for your suggesting -- experimenting with waypoints now. But... it's not easy to move around in Cardboard! Almost impossible. Seems like sometimes teleporting works, other times  now. Mostly you are frozen upon entering your spawnpoint, so we can just place the sphere around that point. But it would be VERY nice to be able to move around in Cardboard :)

**Reply by emclaren** [reply]
typically with cardboard you are frozen where you load. We recommend placing spawnpoints where you want folks to land in the world.

**Comment by InfiniteLee** [comment]
Is the issue you're running into that on cardboard you can only teleport, and instead of teleporting into the photosphere you're clicking on the sphere itself? This can be avoided in spoke by unchecking the "controls" checkbox on the media element. Please let us know if that resolves the issue.

![photosphere](https://user-images.githubusercontent.com/1821537/90699478-35fc6200-e238-11ea-8f1a-8aee27efb466.png)

**Reply by Aaron-E-Walsh** [reply]
Thank you! I will try this. From my other reply to my question you might have read what I wrote about navigating in Cardboard, It's not easy to move around in Cardboard! Almost impossible. Seems like sometimes teleporting works, other times not. Mostly you are frozen upon entering your spawnpoint, so we can just place the sphere around that point. But it would be VERY nice to be able to move around in Cardboard :)

**Reply by emclaren** [reply]
how are you currently moving around in cardboard @Aaron-E-Walsh? 

Also, if it's helpful at all, we've just added a new feature so that you can spawn into a specific spawnpoint in Hubs by adding a little bit of additional text at the end of the URL https://github.com/mozilla/hubs/pull/2837

The format is url#[spawn_point_name]

**Comment by Aaron-E-Walsh** [comment]
Ah, thank you!

We have only been teleporting (by touching the screen using the Cardboard
trigger), so spawnpoints might help -- especially being able to jump
directly to any one of them as you have suggested (great feature, by the
way) and also adding spawpoints for those entering the world for the first
time

Thanks again,
Aaron

**Reply by emclaren** [reply]
No problem! hopefully this helps!

**Comment by kristinlohwasser** [comment]
Hi Aaron, 

I hope you are still following this thread -- we are trying to do something very similar it seems, but for us clicking somewhere on the screen or moving at all in google cardboard is impossible. 

So I wanted to ask you a bit more about your setup and possible exchange ideas/tricks. 

Thanks & Best
Kristin
